templates/publisher/listing.html

```diff
@@ -303,27 +303,47 @@
               <div class="col-12">
                 <div class="p-notification--negative">
                   <p class="p-notification__response">
-                      <strong>Screenshots:</strong> {{ field_errors['screenshots'] }}
+                      <strong>Media:</strong> {{ field_errors['screenshots'] }}
                   </p>
                 </div>
               </div>
             </div>
           {% endif %}
           <div class="row" data-validation-name="screenshots">
-            <div class="col-12">
+            <div class="col-2">
+              <label>Images:</label>
+            </div>
+            <div class="col-8">
+              <div class="p-notification--caution js-media-limit-warning{% if screenshot_urls|length < 5 %} u-hide{% endif %}">
+                <p class="p-notification__response">You have over 5 images uploaded. Not all image will be visible to users.</p>
+              </div>
+              <div class="p-listing-images p-fluid-grid js-media-holder">
```

Comment:
Seems like we are rendering this both in template and in JS.
Any reason for that? I can get that it will make it render without JS as well, and will probably appear earlier on the page before React re-renders it. But the functionality wouldn't work without JS anyway. None of this form would. So I'm not sure it's worth duplicating in template and React. With any change to class names or markup they can get out of date here.

Reply:
Interesting. I just tried without JS and you're right. I thought we still saved title etc. without JS, obviously not! I guess we rely on the "diff" input being submitted.
It would probably be better to have the diff input list which input's have changed, and use that list to construct the POST to dashboard. If ! diff list we should POST all fields.

Reply:
I think the plan was to move it all to React/JS eventually, so having a fallback support for pure POST was not needed.
As much as I like the idea of progressive enhancement and having things working without JS in this case (and with moving much of rendering to React) it may not be worth implementing and maintaining.